Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8.3-alpine3.11
+
+RUN apk --update add --no-cache g++
```

Comment:
Do these 3 lines in a single run, using && after each command, like you did on the 2nd one. That saves space. You can break lines with `\`.

Reply:
Moved to into newline. However, even with the same line it worked.

Reply:
Yes, it will work, but doing 3 lines separately results in a heavier container if there are clean up steps.